### PR TITLE
[Edit] Go: Maps

### PR DESCRIPTION
**Pull request by caupolicandiaz**
### Description

This PR makes several corrections and additions to the Go map entry including:
- Clarifying the declaration syntax which referred to `create` in lieu of `make` which is the syntax used in an example
- Modifying header capitalization
- Adding a section on removing items
- Truncating code examples that use the same map
- Changing the final example to be a codebyte to match the existing header

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x ] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [x ] My changes generate no new warnings.
- [x ] I have performed a self-review of my own writing and code.
- [x ] I have checked my entry and corrected any misspellings.
- [x ] I have made corresponding changes to the documentation if needed.
- [x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/go/concepts/map/map.md [hunk 1]
```diff
@@ -226,11 +204,33 @@ map[Ana:95.6 John:85.2]
...
+  fmt.Println(gradebook)
+
+```
+
+This will output:
+
+````shell
+map[Ana:95.6 Emma:90 George:76.4 John:85.2]
+map[Ana:95.6 Emma:90 George:76.4]
+
```
Comment (yangc95) [inline content/go/concepts/map/map.md:228]:
```suggestion
```

```

Comment (yangc95) [inline content/go/concepts/map/map.md:228]: Closing backticks needed

**Review by yangc95 — APPROVED** [review]
@caupolicandiaz two minor format suggestions. Looks great and you definitely improved clarity for this entry!

File: content/go/concepts/map/map.md (hunk 1 above)
Comment (SSwiniarski) [inline content/go/concepts/map/map.md:228]:
````suggestion
```

````

File: content/go/concepts/map/map.md [hunk 2]
```diff
@@ -269,20 +269,13 @@ func main() {
...
-m with 2 added key-value pairs:  map[Alex:92.5 Amanda:85.2]
-map gradebook:  map[Ana:95.6 John:85.2]
-grade of Bob who does not exist within map gradebook:  0
-Ana grade:  95.6
-map gradebook with 2 more values: map[Ana:95.6 Bob:100 Elizabeth:88.6 John:85.2]
-```
+  // Output map
+  fmt.Println("final map: ", gradebook)
+}
+````
```
Comment (SSwiniarski) [inline content/go/concepts/map/map.md:281]:
````suggestion
```
````

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@caupolicandiaz other than my two comments, looks good to go.

File: content/go/concepts/map/map.md [hunk 3]
```diff
@@ -11,26 +11,36 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-A **map** is a built-in data structure that is used to store a collection of unordered key-value pairs that can be the same or different data type. It is Go's implementation of a hash table, which allows for efficient access, insertion, and deletion.
+A **map** is a built-in data structure that is used to store a collection of unordered key-value pairs. The pairs can be of the same type, or of mixed types. It is Go's implementation of a hash table, which allows for efficient access, insertion, and deletion.
```
Comment (HishamT) [inline content/go/concepts/map/map.md:14]:
```suggestion
A **map** is a built-in data structure that is used to store a collection of unordered key-value pairs. The pairs can be of the same type or of mixed types. It is Go's implementation of a hash table, which allows for efficient access, insertion, and deletion.
```

**Review by HishamT — COMMENTED** [review]
Finished reviewing the entry. I noticed that the functions are written as `function()` as opposed to `.function()`. I believe the standard was to use the '.' in the function name.

**Comment by caupolicandiaz** [comment]
> Finished reviewing the entry. I noticed that the functions are written as `function()` as opposed to `.function()`. I believe the standard was to use the '.' in the function name.

I believe we generally format this based on how the functions are implemented. Some functions are applied using "dot notation" and others like the built-in len() function in python are passed arguments, so we don't include a period.

**Review by SSwiniarski — APPROVED** [review]

**Comment by yangc95** [comment]
> > Finished reviewing the entry. I noticed that the functions are written as `function()` as opposed to `.function()`. I believe the standard was to use the '.' in the function name.
> 
> I believe we generally format the names based on how the functions are implemented. Some functions are applied using "dot notation" and others like the built-in len() function in python are passed arguments, so we don't include a period.

Sounds right! Functions are written without the dot, while methods are. This is dependent on the concept entry 🙂

**Comment by yangc95** [comment]
🎉  @caupolicandiaz edits will appear here: https://www.codecademy.com/resources/docs/go/map

cc: @HishamT @SSwiniarski

**Pull request merged**